kyototycoon/datadog_checks/kyototycoon/kyototycoon.py

```diff
@@ -74,15 +70,15 @@ def check(self, instance):
             r.raise_for_status()
         except requests.exceptions.HTTPError as e:
             self.service_check(self.SERVICE_CHECK_NAME, AgentCheck.CRITICAL, tags=service_check_tags,
-                               message=str(e.message))
+                               message=str(e))
             raise
         except Exception as e:
             self.service_check(self.SERVICE_CHECK_NAME, AgentCheck.CRITICAL, tags=service_check_tags, message=str(e))
             raise
         else:
             self.service_check(self.SERVICE_CHECK_NAME, AgentCheck.OK, tags=service_check_tags)
 
-        body = r.content
+        body = r.text
```

Comment:
why did you wanna change this?

Reply:
r.text is the content of the response in unicode, and r.content is in bytes. There are number of parts of this check that compare that response with strings. Ensuring that the response is unicode allows for the minimal amount of changes to the rest of the check. Otherwise we're comparing bytes to unicode and Python3 throws errors.